githubpoxy.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"errors"
+	"io"
+	"time"
+
+	"github.com/go-rod/rod"
+)
+
+type GitHubProvider struct {
+}
+
+func (ghp GitHubProvider) Login(login, password string) error {

```

Comment:
I usually default to using pointer receivers, i.e. `ghp *GitHubProvider`, but if you plan on storing state between method calls through a `*rod.Browser` you'll be okay.